app/services/user_alerts/alert_user_about_account_verified.rb

```diff
@@ -0,0 +1,16 @@
+module UserAlerts
+  class AlertUserAboutAccountVerified
+    def self.call(user:, date_time:, app:, disavowal_token:)
+      app = app || APP_NAME
```

Comment:
supernit: save some characters:

```suggestion
      app ||= APP_NAME
```